components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -217,6 +243,13 @@ AdsServiceImpl::AdsServiceImpl(Profile* profile,
       display_service_(NotificationDisplayService::GetForProfile(profile_)),
       rewards_service_(
           brave_rewards::RewardsServiceFactory::GetForProfile(profile_)),
+#if BUILDFLAG(BRAVE_ADAPTIVE_CAPTCHA_ENABLED)
+      adaptive_captcha_service_(
+          brave_adaptive_captcha::BraveAdaptiveCaptchaServiceFactory::
```

Comment:
pass in BraveAdaptiveCaptcha service to AdsServiceImpl constructor

Reply:
Fixed.